docs/dsl_resource.md

```diff
@@ -98,3 +98,23 @@ end
 ```
 
 For a full example, see our [example resource](https://github.com/chef/inspec/blob/master/examples/profile/libraries/gordon_config.rb).
+
+## Lazy Loading
```

Comment:
Consider renaming section "Overriding core resources and Lazy Loading"

Reply:
I added a sub-section to split them out.